test/model/model_test.rb

```diff
@@ -153,6 +168,23 @@ class Event
         assert_equal 'text/plain', event.send(:__robject).content_type
         Event.content_type 'application/json'
       end
+
+      test 'persist the object with quorum options' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.